internal/service/ec2/ec2_launch_template_test.go

```diff
@@ -2958,7 +2958,7 @@ func TestAccEC2LaunchTemplate_metadataOptions(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "metadata_options.0.http_endpoint", "enabled"), //Setting any of the values in metadata options will set the http_endpoint to enabled, you will not see it via the Console, but will in the API for any instance made from the template
 					resource.TestCheckResourceAttr(resourceName, "metadata_options.0.http_tokens", "required"),
 					resource.TestCheckResourceAttr(resourceName, "metadata_options.0.http_put_response_hop_limit", "2"),
-					resource.TestCheckResourceAttr(resourceName, "metadata_options.0.http_protocol_ipv6", "disabled"),
+					resource.TestCheckResourceAttr(resourceName, "metadata_options.0.http_protocol_ipv6", "enabled"),
```

Comment:
Since test case is for checking whether it works fine by not setting http_endpoint to true, and we have set computed for http_protocol_ipv6 to true, by default for http_protocol_ipv6 , the value is set to 
"enabled". Verified by removing instance_metadata_tags as well and tests passed (Value expected was enabled)